Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -481,7 +481,10 @@ void CGameListCtrl::InsertItemInReportView(long _Index)
 		}
 	}
 
-	SetItem(_Index, COLUMN_TITLE, StrToWxStr(name), -1);
+	if (rISOFile.IsDiscTwo() && !name.find("Disc 2", 0))
```

Comment:
`name.find("Disc 2", 0) == std::string::npos` the returned result for "not found" is the constant `-1` so comparing against zero is incorrect